privacyidea/lib/resolvers/SQLIdResolver.py

```diff
@@ -449,17 +383,17 @@ def getUserList(self, searchDict=None):
             column = self.map.get(key)
             value = searchDict.get(key)
             value = value.replace("*", "%")
-            conditions.append(getattr(self.TABLE, column).like(value))
+            conditions.append(self.TABLE.columns[column].like(value))
 
         conditions = self._append_where_filter(conditions, self.TABLE,
                                                self.where)
         filter_condition = and_(*conditions)
 
-        result = self.session.query(self.TABLE).\
-            filter(filter_condition).\
-            limit(self.limit)
+        result = self.session.execute(select(self.TABLE).
```

Comment:
Is using `execute` OK or could it be prone to security issues?

Reply:
This is the [SQLAlchemy 2.0 style](https://docs.sqlalchemy.org/en/14/glossary.html#term-2.0-style), so basically for the future :-)